README.md

```diff
@@ -83,6 +83,19 @@ of options:
  * [Libvirt][libvirt]
  * [Cockpit][cockpit]
 
+## Submitting patches
+
+When sending patches to the project, the submitter is required to certify that
+they have the legal right to submit the code. This is achieved by adding a line
+
+    Signed-off-by: Real Name <email@address.com>
```

Comment:
It isn't auto-generated in the way as an @author metadata is. @author is added by git to everything, with no input from the use. The Signed-off-by tag it is an explicit thing users have to consciously add to indicate their acceptance of the developer certificate of origin. Git merely makes it easier to add by giving the "-s" command line flag to insert it in the correct format.

Essentially by adding the Signed-off-by tag, the user is explicitly asserting that they have the legal right to submit the code (either because they are the copyright holder, or because the copyright holder has given them permission).

This is a lightweight approach to legally certifying contributions, which avoids the pain of forcing users to sign a contributor agreement / copyright assignemt for a project. The doc added here is the standard text used the Linux kernel, and many other open source projects (Docker, Chef, OpenDaylight, Eclipse and more).